pkg/event/bus_test.go

```diff
@@ -15,11 +16,11 @@ import (
 type runner struct {
 }
 
-func (r *runner) ExecuteStdIn(cmd string, args []string, env map[string]string, stdin io.Reader) ([]byte, error) {
```

Comment:
Sorry for the radio silence! I'm reviewing this today. Although almost everything looks awesome, I have one enhancement idea-

Can we safely move the ctx to the runner struct field, rather than adding the ctx parameter? That way we can avoid changing every call to ExecuteStdIn and Exexcute.

We already have the ctx in the helmExecer field, instead of adding the ctx parameters to all the helmExecer methods. It should e feasible todo the same for `runner`, assuming the life time of a `runner` would be same as its caller/parent, `helmExecer`. WDYT? 🤔

Reply:
// I'll try changing it this way myself when I have some time.

Reply:
Such change seems to be reasonable. If I right remember I haven't done that just because I haven't found a constructor that would allow me to do that

Reply:
Done. But I ignored the bus.go codepath where runner is initialised separately for the hooks if it wasn't passed